lib/config/config-file.js

```diff
@@ -227,7 +246,7 @@ function loadConfigFile(filePath) {
 function writeJSONConfigFile(config, filePath) {
     debug("Writing JSON config file: " + filePath);
 
-    var content = JSON.stringify(config, null, 4);
+    var content = stringify(config, {cmp: sortByKey, space: 4});
```

Comment:
The sort is to alphabetize the rules, since there are 200 of them, and by default they will come out in a random-ish order, making it difficult to find anything without searching within your config.  The dependency was already being used elsewhere, so it's not adding anything new to to our package.json.